README.md

```diff
@@ -0,0 +1,103 @@
+LTI Learner Profile Dashboard
+=============================
+
+Prerequisites
```

Comment:
Isn't also running MySQL a prerequisite?

Reply:
@tomaszgy Not really, if you don't add the [`DATABASES`](https://github.com/open-craft/learner-profile-dashboard/blob/basic-implementation/README.md#L31-L38) settings mentioned below to `app/local_settings.py`, the app will just use `sqlite`. But you're right that it's worth touching on this here, I went ahead and updated this section.